packages/web/src/components/pages/WebsiteHostnamePage.tsx

```diff
@@ -63,7 +67,19 @@ export default function WebsiteHostnamePage() {
     return <Navigate replace to='/' />;
   }
 
-  if (isInvalidResult) {
+  if (isProtected) {
+    return (
+        <ErrorLayout
+            message="It looks like the entered website is protected with anti-fraud, so we couldn't get script content."
```

Comment:
The entered website is protected by a third-party service, such as DDoS prevention, password protection or geolocation restrictions.

Reply:
Fixed